content/tutorials/esc-setup-psp-creds/index.md

```diff
@@ -0,0 +1,128 @@
+youll_learn:
+    - How to store and retrieve configuration values with Pulumi ESC
+    - How to configure Pulumi Service Provider
+    - How to use Pulumi Service Provider to create more ESC Environments
+prereqs:
+    - Basic understanding of Pulumi IaC the [Pulumi CLI](/docs/iac/download-install/) installed
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - Familiarity with TypeScript and Node.js
+estimated_time: 10
+collections:
+    - pulumi-esc
+---
+
+{{% notes %}}
+This tutorial is all about using ESC with Pulumi Service Provider (PSP for short!). However, using ESC is an encouragement rather than a requirement. If you *don't* want to use ESC, refer to [PSP setup docs](/registry/packages/pulumiservice/installation-configuration/) instead.
+{{% /notes %}}
+
+## Log into Pulumi Cloud
+
+Before you begin, make sure you've [signed into Pulumi Cloud](https://app.pulumi.com/). Once you've done so, you can log in with Pulumi CLI and be prompted to :
+
+```bash
+$ pulumi login
+
+Manage your Pulumi stacks by logging in.
+Run `esc login --help` for alternative login options.
+Enter your access token from https://app.pulumi.com/account/tokens
+    or hit <ENTER> to log in using your browser
+```
+
+## Obtain a personal access token
+
+Next, you'll also need a Pulumi Cloud [personal access token](/docs/pulumi-cloud/access-management/access-tokens/#personal-access-tokens). Create a short-lived access token for this tutorial, copy it into your current shell to login into Pulumi Cloud. Keep the token creation page open in a separate tab, or paste it somewhere temporarily to use in the next step of the tutorial.
+
+## Brief Explanation
+
+Pulumi Service Provider is used to create Pulumi Cloud resources using Pulumi IaC Programs. Just like an S3 bucket is created using an AWS provider, PSP allows creation of things like Environments, Environment Version Tags, Environment Webhooks, and much more. In order to authenticate with Pulumi Cloud, PSP uses the same personal access token (PAT for short!) as Pulumi CLI. This is where ESC comes in!
+
+Instead of providing PAT as an environment variable, risking losing or leaking it, we will create an ESC Environment and store the PAT there. We will then import this environment into the configuration of a new stack, in which we will create Pulumi Cloud resources
+
+## Create a new ESC project and environment
+
+In ESC, configuration settings belong to collections called [_environments_](/docs/esc/environments/working-with-environments/), and environments in turn belong to _projects_. Projects and environments may be created either with the Pulumi Cloud console or with the CLI.
```

Comment:
ESC [Environments](/docs/esc/environments/working-with-environments/) are a great way to store configuration settings and secrets. Environments can be created in the Pulumi Cloud console or with the CLI.